test/components/patch_serviceaccount_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet
+// +build e2e
+
+//nolint
+package components
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	_ "github.com/kinvolk/lokomotive/pkg/components/flatcar-linux-update-operator"
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	retryInterval  = time.Second * 5
+	timeout        = time.Minute * 5
+	contextTimeout = 10
+)
+
+type componentTestCase struct {
+	namespace string
+	platforms []testutil.Platform
+}
+
+func TestPatchServiceAccount(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+	componentTestCases := []componentTestCase{
+		{
```

Comment:
Can we go through every namespace instead of explicitly listing them here?

Reply:
We have to check components for the different platforms so I don't know that would be feasible because **AKS** only has **5 components** and **AWS** and **AWSEdge** has 14 and Packet has 15. Now we are checking for all the platforms so, it might be difficult. But if you can suggest alternate solution I am happy to change it

Reply:
You can simply enlist all the namespaces on the cluster and verify? To disable checks for special namespaces like `kube-system` just remove it. Like maintain a `blocklist` for the namespaces not to test instead of `safelist` like it is done right now.

Reply:
Aaah, I forgot about the API, so silly of me